terraform/cyhy_private_dns.tf

```diff
@@ -161,17 +176,26 @@ resource "aws_route53_zone" "cyhy_private_zone_reverse" {
 
 resource "aws_route53_record" "cyhy_rev_reporter_PTR" {
   zone_id = "${aws_route53_zone.cyhy_private_zone_reverse.zone_id}"
-  name    = "${local.the_reporter}.${aws_route53_zone.cyhy_private_zone_reverse.name}"
+  name    = "${format("%s.%s.%s.%s.in-addr.arpa.",
+    element(split(".", aws_instance.cyhy_reporter.private_ip) ,3),
+    element(split(".", aws_instance.cyhy_reporter.private_ip) ,2),
+    element(split(".", aws_instance.cyhy_reporter.private_ip) ,1),
+    element(split(".", aws_instance.cyhy_reporter.private_ip) ,0),
+  )}"
   type    = "PTR"
   ttl     = 300
   records = [ "reporter.${local.cyhy_private_domain}." ]
 }
 
 resource "aws_route53_record" "cyhy_rev_database_PTR" {
-  count = "${local.count_database}"
   zone_id = "${aws_route53_zone.cyhy_private_zone_reverse.zone_id}"
-  name    = "${local.first_database + count.index}.${aws_route53_zone.cyhy_private_zone_reverse.name}"
+  name    = "${format("%s.%s.%s.%s.in-addr.arpa.",
+    element(split(".", aws_instance.cyhy_mongo.private_ip) ,3),
+    element(split(".", aws_instance.cyhy_mongo.private_ip) ,2),
+    element(split(".", aws_instance.cyhy_mongo.private_ip) ,1),
+    element(split(".", aws_instance.cyhy_mongo.private_ip) ,0),
+  )}"
   type    = "PTR"
   ttl     = 300
-  records = [ "database${count.index + 1}.${local.cyhy_private_domain}." ]
+  records = [ "database1.${local.cyhy_private_domain}." ]
```

Comment:
What happened to our multiple database support?

Reply:
It's back.